src/controllers/rooms.controller.js

```diff
@@ -0,0 +1,73 @@
+const { Room } = require('../models/room.model');
+const roomsService = require('../services/rooms.service');
+const { ApiError } = require('../utils/api.error');
+
+const getAll = async (req, res) => {
+  const rooms = await roomsService.getAll();
+
+  res.send(rooms);
+};
+
+const create = async (req, res) => {
+  const { name, userId } = req.body;
+
+  if (!name || !userId) {
+    throw ApiError.badRequest('Not all data is provided!');
```

Comment:
The error message 'Not all data is provided!' could be more specific to indicate which data is missing. Consider specifying the missing fields in the error message.